src/applications/facility-locator/constants/index.js

```diff
@@ -31,6 +32,17 @@ export const FacilityType = {
   VET_CENTER: 'vet_center',
 };
 
+/**
+ *
```

Comment:
Should we remove this empty comment?

Reply:
Hah! I meant to actually leave a comment about this block, missed it in my rush. Thanks 👍